modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/index.d.ts

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+interface CommerceAccount {
+	id: string;
+	name: string;
+}
+
+interface CommerceOrder {
+	id: string;
+	name: string;
+}
+
+interface CommerceCurrency {
+	code: string;
+	id: string;
+	name: string;
+}
+
+interface CommerceContext {
+	accountEntryAllowedTypes: string[];
+	commerceAccount: CommerceAccount;
```

Comment:
Is the `commerce` prefix needed? Since we are already at `CommerceContext`... :thinking:

Reply:
I wanted to try and keep the naming as close to the CommerceContext class since that is what defines these values.